packages/admin/cms-admin/src/dam/FileForm/FileSettingsFields.tsx

```diff
@@ -111,6 +146,19 @@ export const FileSettingsFields = ({ isImage, folderId }: SettingsFormProps): Re
                     name="title"
                     component={FinalFormInput}
                     fullWidth
+                    endAdornment={
+                        ContentGenerationEnabled && (
+                            <IconButton
+                                sx={{ color: theme.palette.primary.main }}
+                                onClick={async () => {
+                                    const { data } = await generateImageTitle({ variables: { imageUrl: file.fileUrl } });
```

Comment:
I'm not sure if this will work in a deployed setting. We use preview URLs everywhere meaning they are routed through the auth proxy. I'm not sure if the API can access files behind the auth proxy (@johnnyomair Can you answer this?). If not, the API can't "download" this file to convert it into a base64 string. 

Instead we could just pass the id and create a non-preview URL in the API

Reply:
You're right, that won't work.

> Instead we could just pass the id and create a non-preview URL in the API

Either this, or we could read the file directly from the blob storage backend and convert it to base64 in the API